CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # v1.16
 
+## Release v1.16.6 - 2024/08/16
+
+### Bug Fix
+
+* YAML config syntax: Fix issue where `$log_level` element was not supported correctly
+  https://github.com/fluent/fluentd/pull/4486
+* out_file: Add warn message for symlink_path setting
+  https://github.com/fluent/fluentd/pull/4512
```

Comment:
Should it be Enhancement or Misc category?

Reply:
Oh, I see.
Indeed it can be.
It is hard to decide.
Given that it is a Stable branch, I hesitate to add a `Enhancement` element easily, but that might be my overthinking.

#4512 is a measure to deal with an undesirable behavior.
From that perspective, it would be more like `Bug Fix` than `Enhancement`.

However, #4512 did not fix the behavior itself.
From that perspective, it can be `Misc` or `Enhancement`.

What do you think?

Reply:
In this context, Misc may be better.

Reply:
OK! Let's put it in `Misc`!

Reply:
In v1.17.1 side, fixed by https://github.com/fluent/fluentd/pull/4607.

Reply:
Thanks! I have fixed this side.